### PR TITLE
chore: use ipld-explorer-components

### PR DESCRIPTION
**Pull request by olizilla**
The core of the app is extracted to https://github.com/ipfs-shipyard/ipld-explorer-components so we can reuse it from webui

License: MIT
Signed-off-by: Oli Evans <oli@tableflip.io>

### CODE REVIEW

**Comment by hacdias** [comment]
@olizilla 😍 this! It would be reaaaaallly cool to do the same for [WebUI's components directory](https://github.com/ipfs-shipyard/ipfs-webui/tree/revamp/src/components) because those are the ones we can use on all apps like Buttons and such! This is amazing 😄

**Comment by olizilla** [comment]
@hacdias yep, we can figure that out next week once webui is shipped! I'm looking at https://jxnblk.com/rebass for inspiration / the foundation.

File: package.json [hunk 1]
```diff
@@ -59,29 +41,34 @@
...
     "source-map-explorer": "^1.5.0",
     "standard": "^11.0.1"
   },
   "homepage": "./",
   "standard": {
     "parser": "babel-eslint"
   },
+  "browsers": [
+    ">0.25%",
+    "not dead"
```
Comment (lidel) [inline package.json]: :laughing: 

Jokes aside, do we have a policy for this? So far we are quite creative:
- https://github.com/ipfs-shipyard/ipld-explorer-components/blob/cb745c55b9383b5cf394fa15508545b3f7fdc442/package.json#L80
- https://github.com/ipfs-shipyard/peer-pad/blob/e65fa08e6bce4627b58e6e82e0fdc0382a3b828f/config/createStyleRule.js#L28
- https://github.com/ipfs-shipyard/IGiS/blob/6a52b244ee52c294ac1a4c0e2042fae549b9e282/config/webpack.config.dev.js#L179 
- https://github.com/ipfs-shipyard/ipfs-share-files/blob/8217ffd4a4712e2175d4e3567a23108a8f58960d/config/webpack.config.prod.js#L226
- and we may have duplicate in this very repo: https://github.com/ipfs-shipyard/ipld-explorer/blob/ea4b8ccab65d964f2cc4fd57e4c93b21bfe6683d/config/webpack.config.prod.js#L226-L230

**Comment by olizilla** [comment]
Now in i18n flavour

screenshot 2018-09-11 18 05 40

**Comment by hacdias** [comment]
@olizilla this is really cool. I've just updated the translations here and on ipfs-shipyard/ipld-explorer-components. Added the new ones and fixed one typo I noticed on that screenshot!

File: src/i18n.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import XHR from 'i18next-xhr-backend'
+import LanguageDetector from 'i18next-browser-languagedetector'
+
+import en from 'i18next-icu/locale-data/en'
+import pl from 'i18next-icu/locale-data/pl'
+import pt from 'i18next-icu/locale-data/pt'
+
+i18n
+  .use(new ICU({
+    localeData: [en, pl, pt]
```
Comment (lidel) [inline src/i18n.js]: Missing `sv` :)
(also 4 lines before, add `import sv from 'i18next-icu/locale-data/sv'`

File: package.json (hunk 1 above)
Comment (olizilla) [reply package.json]: We currently have no best practice for this. I was persuaded by https://jamie.build/last-2-versions that the old "last 2 versions" probably wasn't what we wanted.

"not dead" is now part of a recommended browser list, where `dead` is defined as

- `dead`: browsers from `last 2 version` query, but with less than 0.5%
  in global usage statistics and without official support or updates
  for 24 months. Right now it is `IE 10`, `IE_Mob 10`, `BlackBerry 10`,
  `BlackBerry 7`, and `OperaMobile 12.1`.

but the current browserlist defaults do look sensible

- * `defaults`: Browserslist’s default browsers
  (`> 0.5%, last 2 versions, Firefox ESR, not dead`).

**Pull request merged**